python/cudf/cudf/core/column/datetime.py

```diff
@@ -589,6 +601,18 @@ def as_string_column(
     ) -> "cudf.core.column.StringColumn":
         return self._local_time.as_string_column(dtype, format, **kwargs)
 
+    def __repr__(self):
```

Comment:
This is really unrelated to the rest of the PR, but a quality-of-life thing for debugging. Pandas always prints the _local_ timestamps when looking at a tz-aware column and pyarrow always prints the UTC timestamps.